.github/workflows/ci.yml

```diff
@@ -14,24 +14,30 @@ concurrency:
 jobs:
   drupal_codequality:
     runs-on: ubuntu-latest
+    container: hussainweb/drupalqa
     steps:
       - name: Check out repository code
         uses: actions/checkout@v4
 
-      - name: Drupal Code Quality
-        uses: hussainweb/drupalqa-action@v2
-        id: drupalqa
-        with:
-          php-version: 8.2
-          checks: |
-            phplint: {}
-            phpcs:
-              standard: phpcs.xml
-              ignore: /node_modules/
-            phpmd:
-              ruleset: phpmd.xml
-              suffixes: php
+      - name: Enable GrumPHP plugin
+        run: composer global config --no-plugins allow-plugins.phpro/grumphp-shim true
+
+      - name: Install GrumPHP globally
+        run: composer global require phpro/grumphp-shim
+
```

Comment:
```suggestion
```

I just pushed a new Docker image with GrumPHP built-in. I will also make it a specific check soon.

Reply:
Noted